ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/QueryFeaturesWithArcadeExpression/README.md

```diff
@@ -0,0 +1,49 @@
+# Query features with arcade expression
+
+Query features on a map using an arcade expression.
+
+![](screenshot.png)
+
+## Use case
```

Comment:
```suggestion
## Use case

```
Blank lines should follow headers. Otherwise this sometimes breaks markdown renderers and causes the entire block to be a header.